trust/meson.build

```diff
@@ -0,0 +1,165 @@
+libtrust_data_sources = [
+  'asn1.c', 'asn1.h',
+  'base64.c', 'base64.h',
+  'pem.c', 'pem.h',
+  'oid.c', 'oid.h',
+  'utf8.c', 'utf8.h',
+  'x509.c', 'x509.h'
+]
+
+asn_sources = [
+  'basic.asn',
+  'pkix.asn',
+  'openssl.asn'
+]
+
+asn_h_deps = []
+foreach name : asn_sources
+  asn_h = custom_target('generate @0@.h'.format(name),
```

Comment:
I've never used it properly but what about using [generator](http://mesonbuild.com/Reference-manual.html#generator) here?

Reply:
Yes, generator fits well in this use-case.